structure_tests/README.md

```diff
@@ -127,6 +125,20 @@ globalEnvVars:
 ```
 
 
+## Running File Tests Without Docker
+
+Container images can be represented in multiple formats, and the Docker image is just one of them. At their core, images are just a series of layers, each of which is a tarball, and so can be interacted with without a working Docker daemon. While running command tests currently requires a functioning Docker daemon on the host machine, File Existence/Content tests do not. This can be particularly useful when dealing with images which have been `docker export`ed or saved in a different image format than the Docker format. To run tests without using a Docker daemon, a user can specify a different "driver" to use in the tests, with the `-driver` flag.
```

Comment:
Nit: A lot of markdown linters prefer 80 character lines. I usually break at 80 and put each sentence on a new line. It helps with readability in an editor.